### PR TITLE
feat: add `llm` module to interact with LLMs from LiteLLM

### PR DESCRIPTION
**Pull request by rodonguyen**
- Add `llm` module to interact with LLMs via LiteLLM
- Use `llm` to send request to LLM in `discord_bot` 
- Update `.env.development`

### CODE REVIEW

File: src/discord_bot/bot.py [hunk 1]
```diff
@@ -42,18 +35,21 @@ async def my_long_command_function(ctx: interactions.SlashContext) -> None:
   name="prompt", description="Enter your prompt", required=True, opt_type=interactions.OptionType.STRING, min_length=10
 )
 async def ask_model(ctx: interactions.SlashContext, model: str = "", prompt: str = "") -> None:
-  await ctx.send(f'You asked model {model} with the prompt: "{prompt}"')
+  if model not in MODEL_CHOICES:
```
Comment (nqngo) [inline src/discord_bot/bot.py:38]: Is this covered by https://github.com/bifrostlab/llm-assistant/pull/32 already. Don't pull from different PR in before merging.

**Review by nqngo — CHANGES_REQUESTED** [review]

File: src/discord_bot/bot.py (hunk 1 above)
Comment (rodonguyen) [reply src/discord_bot/bot.py:38]: That's why I put it in Draft mode sir

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,6 @@
+AI_SERVER_URL="http://localhost:8000"
 DISCORD_BOT_TOKEN="EXAMPLETokenTTTTT.Grz0fK.IUi1IONPcLHZsnyjROTt8lR2fuYCeK4grzRoSQ"
+OPENAI_API_KEY="ee-example8888888888888888" # needed for running litellm
 
 # THIS SHOULD BE ASSIGNED TO THE BOT'S `scopes` WHEN DEPLOYED
 DISCORD_GUILD_ID="99999999999999999999999"
```
Comment (samhwang) [inline .env.example:6]: I would recommend lumping all Discord related envs together, and all LiteLLM/OpenAI/... things together.

```suggestion
# LiteLLM/OpenAI
AI_SERVER_URL="http://localhost:8000"
OPENAI_API_KEY="ee-example8888888888888888" # needed for running litellm

# Discord
DISCORD_BOT_TOKEN="EXAMPLETokenTTTTT.Grz0fK.IUi1IONPcLHZsnyjROTt8lR2fuYCeK4grzRoSQ"
# THIS SHOULD BE ASSIGNED TO THE BOT'S `scopes` WHEN DEPLOYED
DISCORD_GUILD_ID="99999999999999999999999"
```

Comment (samhwang) [reply .env.example:6]: My second question is, if `OPEN_API_KEY` isn't neede for LiteLLM (as we only need a fake key), why don't we just hard code it in the code?

Comment (rodonguyen) [reply .env.example:6]: @samhwang we still need to assign it to ENV before running LiteLLM

in `src/litellm/README.md`:
```shell
# if you use OpenAI models
export OPENAI_API_KEY=<your_key>
litellm --config llm_assistant/ollama/proxy_config.yaml
```

Comment (samhwang) [reply .env.example]: What I mean is you won't need it in the code, esp. if you put an export

**Review by samhwang — APPROVED** [review]
